### ISSUE TITLE
%Q30 statistic for fastqc module

### ISSUE THREAD

**Issue by TMiguelT** [ewels/MultiQC#1384]
**Is your feature request related to a problem? Please describe.**

Currently, the scalar values (ie stored in the general stats section) produced by the fastqc module don't include anything relating to quality scores. This is unfortunate, because this is quite crucial information. We do have the plots, which is great, but the general stats values are much more helpful for downstream analysis a la MegaQC.

**Describe the solution you'd like**

It is possible to calculate the %Q30 (proportion of bases with quality score equal to or above 30) from the fastqc output. This can then be added to the general stats section.

**Describe alternatives you've considered**

We could also have something like mean/median/some quantiles/stdev of quality scores. But it seems to me that Q30 is more commonly used. These might also be useful, though

**Comment by ewels** [comment]
Yup, could be nice - especially if with a config option for the module where you can set what quality threshold is used to calculate this (or multiple?). Defaulting to 30.

**Comment by TMiguelT** [comment]
Just wondering if there are any tests for the fastqc parsing that I could use to develop this?

**Comment by ewels** [comment]
Tests like unit tests you mean? Not really. There is test data that the CI uses (and I use when reviewing PRs): https://github.com/ewels/MultiQC_TestData/tree/master/data/modules/fastqc
